### PR TITLE
Add instructions for running local perf tests

### PR DESCRIPTION
**Pull request by DrewScoggins**
Adding instructions for running the performance tests locally.

### CODE REVIEW

**Comment by DrewScoggins** [comment]
PTAL @dotnet/rap-team

**Comment by jorive** [comment]
LGTM

File: Documentation/project-docs/performance-guidelines.md [hunk 1]
```diff
@@ -50,5 +50,14 @@ Capturing a trace using PerfView will allow you to:
...
 
+# Running the CoreCLR Performance Tests on Windows#
+1. The first step to running the performance tests locally is to do a release build of CoreCLR and all of the performance tests.  You can do this with the command `build.cmd x64 Release`, this will of course build the x64 runtime, and you should use x86 if you want to test x86.
+
+2. After building the runtime you will need to generate a core root that contains all of the binaries we just built along with the required dependencies.  This can be done with the command `tests\runtest.cmd Release x64 GenerateLayoutOnly`, with the same caveat that x86 should be used if that is the platform that you are testing.
+
+3. Now we need to actually run the performance tests.  You can do that with the following command that should be run from the root of your repo `tests\scripts\run-xunit-perf.cmd -arch x64 -configuration Release -testBinLoc bin\tests\Windows_NT.x64.Release\Jit\Performance\CodeQuality`.
```
Comment (adiaaida) [inline Documentation/project-docs/performance-guidelines.md]: May want to add additional details about running individual benchmarks. Something along the times of "run-xunit-perf.cmd will run all the perf benchmarks found in the directory supplied and its subdirectories. If you want to run a single benchmark, provide the path to its parent directory."

Comment (adiaaida) [inline Documentation/project-docs/performance-guidelines.md]: You may also want to point out that run-xunit-perf.cmd needs to be run from the root of coreclr.

Comment (DrewScoggins) [inline Documentation/project-docs/performance-guidelines.md]: I already call this out

> You can do that with the following command that should be run from the root of your repo

Comment (DrewScoggins) [inline Documentation/project-docs/performance-guidelines.md]: Done

Comment (adiaaida) [inline Documentation/project-docs/performance-guidelines.md]: Reading is hard. Sorry.

**Pull request merged**